x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -293,10 +293,46 @@ interface AllowlistFields {
   [key: string]: boolean | AllowlistFields;
 }
 
+// Allow list process fields within events.  This includes "process" and "Target.process".'
+/* eslint-disable @typescript-eslint/naming-convention */
+const allowlistProcessFields: AllowlistFields = {
+  name: true,
+  executable: true,
+  command_line: true,
+  hash: true,
+  pid: true,
+  uptime: true,
+  Ext: {
+    architecture: true,
+    code_signature: true,
+    dll: true,
+    token: {
+      integrity_level_name: true,
+    },
+  },
+  parent: {
+    name: true,
+    executable: true,
+    command_line: true,
+    hash: true,
+    Ext: {
+      architecture: true,
+      code_signature: true,
+      dll: true,
+      token: {
+        integrity_level_name: true,
+      },
+    },
+    uptime: true,
+    pid: true,
+    ppid: true,
+  },
+  thread: true,
```

Comment:
This covers all the fields in `Target.process.thread`, since we were listing every single one anyway.

Reply:
Makes sense. As long as no PII can filter through we can index it.